api/types/types_drivers_executor.go

```diff
@@ -46,6 +46,18 @@ const (
 
 	// LSXCmdMounts is the command for getting a list of mount info objects.
 	LSXCmdMounts = "mounts"
+
+	// LSXCmdVolumeCreate is the command for creating a volume.
+	LSXCmdVolumeCreate = "volumeCreate"
+
+	// LSXCmdVolumeRemove is the command for removing a volume
+	LSXCmdVolumeRemove = "volumeRemove"
+
+	// LSXCmdVolumeAttach is the command for attaching a volume.
+	LSXCmdVolumeAttach = "volumeAttach"
+
+	// LSXCmdVolumeDeatch is the command for detaching a volume.
+	LSXCmdVolumeDeatch = "volumeDetach"
```

Comment:
Ah yes, it clearly does say that. Apologies. That would be the answer to "what am I missing"?  😄